test/files/run/t12276.check

```diff
@@ -0,0 +1,50 @@
+00000000  1b 5b 33 35 6d 0a 73 63  61 6c 61 3e 20 1b 5b 30  |.[35m.scala> .[0|
+00000010  6d 6a 61 76 61 2e 6e 69  6f 2e 43 68 61 72 42 75  |mjava.nio.CharBu|
+00000020  66 66 65 72 2e 61 6c 6c  6f 63 61 74 65 28 35 29  |ffer.allocate(5)|
+00000030  0a 76 61 6c 20 1b 5b 31  6d 1b 5b 33 34 6d 72 65  |.val .[1m.[34mre|
+00000040  73 30 1b 5b 30 6d 3a 20  1b 5b 31 6d 1b 5b 33 32  |s0.[0m: .[1m.[32|
+00000050  6d 6a 61 76 61 2e 6e 69  6f 2e 43 68 61 72 42 75  |mjava.nio.CharBu|
+00000060  66 66 65 72 1b 5b 30 6d  20 3d 20 3f 3f 3f 3f 3f  |ffer.[0m = ?????|
+00000070  0a 1b 5b 33 35 6d 0a 73  63 61 6c 61 3e 20 1b 5b  |..[35m.scala> .[|
+00000080  30 6d 6a 61 76 61 2e 6e  69 6f 2e 43 68 61 72 42  |0mjava.nio.CharB|
+00000090  75 66 66 65 72 2e 61 6c  6c 6f 63 61 74 65 28 36  |uffer.allocate(6|
```

Comment:
The output looks familiar (and nice!) but I feel like I'd find it more useful if it were split by lines before being split by 16 bytes.  Did you do this like so for any reason other than familiarity/similarity?

Reply:
On second thought, I almost appended the text, so you could just glance at the line output.

I did not because the test is just to witness that certain non-printables are present (and others are not).

But maybe outputting line at a time would be easier to examine.